### PR TITLE
Update CODEOWNERS to remove me from most tgui interfaces

### PR DESCRIPTION
**Pull request by stylemistake**
## About The Pull Request

I am disowning `/tgui/packages/tgui/interfaces` because it creates a lot of noise in my notifications (sometimes it's just an update in prefs), and I feel confident code there is totally fine as long as it passes linters.

I also don't need to see what's in `chat-dark/light.scss` for the same reason.

If anyone codes something ambitious in tgui I still wish to be manually added as a reviewer.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -78,10 +78,16 @@
...
+/code/modules/tgchat/ @stylemistake
+/code/modules/tgui/ @stylemistake
+/code/modules/tgui_panel/ @stylemistake
+/tgui/ @stylemistake
+
+# stylemistake (explicitly disowned)
+
+/tgui/packages/tgui/interfaces/
+/tgui/packages/tgui-panel/styles/goon/chat-dark.scss
+/tgui/packages/tgui-panel/styles/goon/chat-light.scss
```
Comment (Mothblocks) [inline .github/CODEOWNERS:90]:
```suggestion
# /tgui/packages/tgui/interfaces/
# /tgui/packages/tgui-panel/styles/goon/chat-dark.scss
# /tgui/packages/tgui-panel/styles/goon/chat-light.scss
```

i think it's a tad bit weird to have these as entries with no owners

Comment (stylemistake) [reply .github/CODEOWNERS:90]: That's just how it works, according to docs.

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax

Comment (Mothblocks) [reply .github/CODEOWNERS:90]: odd

**Comment by Mothblocks** [comment]
updated the title just so it's clearer on the tracker

**Pull request merged**